JamfSync/Multipart/MultipartUpload.swift

```diff
@@ -130,16 +146,12 @@ class MultipartUpload {
 
                 LogManager.shared.logMessage(message: "Upload token time remaining: \(timeLeft) minutes", level: .debug)
                 if timeLeft < 5 {
```

Comment:
Would be a good idea to have a let variable for this "magic" number with a descriptive name so it's easy to change and a little bit self documenting.